services/slack/service.go

```diff
@@ -99,7 +119,8 @@ func (s *Service) Alert(channel, message, username, iconEmoji string, level aler
 	if err != nil {
 		return err
 	}
-	resp, err := http.Post(url, "application/json", post)
+	s.logger.Println("D! Posting event to Slack", post)
+	resp, err := s.client.Post(url, "application/json", post)
```

Comment:
Hum. I'm not a Go dev. This looks like a bit tricky for me.. O_ô 
I will try to do this next tuesday when i go back to work.